apps/webapp-source/README.md

```diff
@@ -0,0 +1,56 @@
+# Web App from source
+
+**NOTE**: support for Go web apps from source is in preview and not supported
+for production apps yet. The feature is hidden behind a flag which the scripts
+here use.
+
+Continuously build and deploy a Go web app using Azure App Service and GitHub,
+no container required.
+
+This package includes a basic Go web app and scripts to set up infrastructure
+for it in Azure.
+
+Requires [Azure CLI][].
+
+## Try It!
+
+1. Copy the contents of this package to a new repo.
+1. Set configuration in a local `.env` file in the package root by copying
+   `.env.tpl` to `.env`.
+1. Change `AZURE_BASE_NAME` to something relatively unique to you; for example
+   you might include your name.
+1. Change `REPO_NAME` to your org and repo in GitHub, e.g. `joshgav/go-sample`.
+1. Run ./[setup.sh][] to set up an App Service web app connected to the
+   specified repo. Don't forget to `git push` your code there too!
+
+**NOTE**: GitHub sync requires a [GitHub personal access
+token](https://github.com/settings/tokens); you need to get one from the linked
+page and set it in a local environment variable `GH_TOKEN`, e.g. `export
+GH_TOKEN=mylongtokenstring`. You can also add it to your local `.env` file for
```

Comment:
Don't suggest doing this, it's a security risk since it publishes an auth token to a public resource.